website/conferences/model.py

```diff
@@ -25,6 +25,19 @@ class Conference(StoredObject):
     public_projects = fields.BooleanField(required=False, default=True)
     poster = fields.BooleanField(default=True)
     talk = fields.BooleanField(default=True)
+    field_names = fields.DictionaryField(
```

Comment:
In retrospect field_names is probably not the most descriptive name for what this does (contains label text for customization). But this would be a mess to refactor now. :)

Can you please add a 1-line comment describing what it does (intended use case etc) to help future developers understand and use it? That should help a lot.